plugins/error-augmentor/plugin.go

```diff
@@ -45,7 +44,6 @@ func NewDefaultPlugin() *ErrorAugmentorPlugin {
 
 func NewPlugin() *ErrorAugmentorPlugin {
 	return &ErrorAugmentorPlugin{
-		properties:          nil,
```

Comment:
@JesseTatasciore This didn't seem to be used. Was this just a left-over?

Reply:
Yeah it must have been